### ISSUE TITLE
Day not present in Date of birth

### ISSUE THREAD

**Issue by PushKK** [PeWu/topola-viewer#27]
I not see Days in my 4-th level and more of ancestors:
all my ancestors before 1918 years of birth not have "Day" of Date of Birth.

On site https://pewu.github.io/topola-viewer/:
Рождение
июнь 1903 г.
(Day not present)

On GRAMPS:
25.06.1903
(Day present)

On GEDCOM:
1 BIRT
2 DATE @#DJULIAN@ 25 JUN 1903
(Day present)

May be problem - in deep level of ancestors,
may be in @#DJULIAN@.

**Comment by PeWu** [comment]
Yes, the problem is with `@#DJULIAN@`. The date parser does not handle calerdar type specification. The parser needs to be updated here: https://github.com/PeWu/topola/blob/master/src/gedcom.ts#L57
